src/main/java/com/avispl/symphony/dal/communicator/aggregator/properties/PropertyNameConstants.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.aggregator.properties;
+
+/**
+ * Properties used in code, to provide more consistency.
+ *
+ * @author Maksym.Rossiytsev
+ * @since 1.0.0
+ */
+public class PropertyNameConstants {
+
+    public static final String ROOM_CONTROLS_MEETING_SETTINGS_GROUP = "RoomControlsMeetingSettings#";
+    public static final String ROOM_CONTROLS_ALERT_SETTINGS_GROUP = "RoomControlsAlertSettings#";
+    public static final String ACCOUNT_CONTROLS_MEETING_SETTINGS_GROUP = "AccountMeetingSettings#";
+    public static final String ACCOUNT_CONTROLS_ALERT_SETTINGS_GROUP = "AccountMeetingSettings#";
+
+    public static final String LEAVE_CURRENT_MEETING_CONTROL = "RoomControls#LeaveCurrentMeeting";
+    public static final String LEAVE_CURRENT_MEETING_PROPERTY = "LeaveCurrentMeeting";
+    public static final String END_CURRENT_MEETING_CONTROL = "RoomControls#EndCurrentMeeting";
+    public static final String END_CURRENT_MEETING_PROPERTY = "EndCurrentMeeting";
+    public static final String RESTART_ZOOM_ROOMS_CLIENT_CONTROL = "RoomControls#RestartZoomRoomsClient";
+    public static final String START_ROOM_PMI_CONTROL = "RoomControls#StartRoomPersonalMeeting";
+
+    public static final String ROOM_USER_DETAILS_PMI = "RoomUserDetails#PMI";
+    public static final String ROOM_USER_DETAILS_GROUP = "RoomUserDetails#";
+
+    public static final String LOCATION_NAME = "Location#Name";
+    public static final String LOCATION_ID = "Location#ID";
+
+    public static final String LOCATION_ID_PROPERTY = "LocationId";
+    public static final String DEVICE_TYPE_PROPERTY = "DeviceType";
+    public static final String APP_VERSION_PROPERTY = "AppVersion";
+    public static final String DEVICE_SYSTEM_PROPERTY = "DeviceSystem";
+    public static final String DEVICE_STATUS_PROPERTY = "DeviceStatus";
+    public static final String ROOM_DEVICES_TEMPLATE_PROPERTY = "RoomDevices%ss#%s";

```

Comment:
Just to make it plural :) 
So first entry would be one of the following: "ZoomRoomsComputer, Controller, SchedulingDisplay, ZoomRoomsControlSystem, CompanionWhiteboard", making the group name to be "RoomDevicesZoomRoomComputer", so adding "s" there to make it plural, since every such group is a summary of all the devices of such type